service/src/main/java/uk/gov/hmcts/reform/fpl/service/cafcass/api/CafcassApiSearchCaseService.java

```diff
@@ -19,18 +23,32 @@
 @Service
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class CafcassApiSearchCaseService {
+    private static final MustNot CASE_STATES = MustNot.builder()
+        .clauses(List.of(
+            MatchQuery.of("state", "Open"),
+            MatchQuery.of("state", "CLOSED"),
```

Comment:
I wonder if with some of the tickets coming our way of opening listing hearings + approving orders in closed cases (getting ahead of the day for refinement later 😆), they may still need updates on closed cases

Reply:
Updated to include Closed case